js/tsconfig.json

```diff
@@ -17,6 +18,7 @@
 			"DOM",
 			"ESNext.String"
 		] /* Specify library files to be included in the compilation. */,
+    "skipLibCheck": true,
```

Comment:
Bad rebase here too, you can delete it 👍

Reply:
Without this line the linter fails, since typescript react-native-reanimated code is a mess and eslint checks the code in node modules